x-pack/plugin/esql/qa/testFixtures/src/main/resources/lookup-join.csv-spec

```diff
@@ -1650,3 +1650,69 @@ event_duration:long
 2764889
 3450233
 ;
+
+
+joinMaskingRegex
```

Comment:
I am not a big fan of having such test queries here. At least, add a comment with the link to the original bug report.